plugins/modules/aci_epg_useg_attribute_simple_statement.py

```diff
@@ -0,0 +1,436 @@
+description:
+- Manage EPG useg Attributes Simple Statements
+options:
+  tenant:
+    description:
+    - The name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  ap:
+    description:
+    - The name of an existing application network profile.
+    type: str
+    aliases: [ app_profile, app_profile_name ]
+  epg:
+    description:
+    - The name of an existing end point group.
+    type: str
+    aliases: [ epg_name ]
+  parent_block_statements:
+    description:
+    - The list of parent block statements.
+    - The order of the provided list matters, assuming the list ["A", "B", "C"].
+    - The block statement "A" will be the parent of "B"
+    - The block statement "A" will be a child of the default block statement.
+    - The maximum amount of parent block statements is 3.
+    type: list
+    elements: str
+    aliases: [ blocks, parent_blocks ]
+  name:
+    description:
+    - The name of the EPG useg attribute.
+    type: str
+    aliases: [ useg_attribute_name ]
+  type:
+    description:
+    - The type of the EPG useg attribute
+    type: str
+    required: true
+    choices:
+    - ip
+    - mac
+    - dns
+    - ad_group
+    - vm_custom_attr
+    - vm_vmm_domain
+    - vm_operating_system
+    - vm_hypervisor_id
+    - vm_datacenter
+    - vm_id
+    - vm_name
+    - vm_folder
+    - vm_folder_path
+    - vm_vnic
+    - vm_tag
+    aliases: [ useg_attribute_type ]
+  operator:
+    description:
+    - The operator of the EPG useg attribute.
+    type: str
+    choices: [ equals, contains, starts_with, ends_with ]
+  category:
+    description:
+    - The name of the vmware tag category or vmware custom attribute.
+    type: str
+    aliases: [ custom_attribute ]
+  use_subnet:
+    description:
+    - Whether to use the EPG subnet definition for ip.
+    type: bool
+  value:
+    description:
+    - The value of the EPG useg attribute.
+    type: str
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+extends_documentation_fragment:
+- cisco.aci.aci
+- cisco.aci.annotation
+
+notes:
+- The I(tenant), I(ap) and I(epg) used must exist before using this module in your playbook.
+  The M(cisco.aci.aci_tenant), M(cisco.aci.aci_ap) and M(cisco.aci.aci_epg) modules can be used for this.
+seealso:
+- module: cisco.aci.aci_tenant
+- module: cisco.aci.aci_ap
+- module: cisco.aci.aci_epg
+- name: APIC Management Information Model reference
+  description: More information about the internal APIC class B(fv:DnsAttr), B(fv:IdGroupAttr), B(fv:IpAttr), B(fv:MacAttr), and B(fv:VmAttr).
+  link: https://developer.cisco.com/docs/apic-mim-ref/
+author:
+- Christian Kolrep (@Christian-Kolrep)
+- Akini Ross (@akinross)
+"""
+
+EXAMPLES = r"""
+- name: Add a new vmtag useg attribute in default block statement
+  cisco.aci.aci_epg_useg_attribute:
```

Comment:
```suggestion
  cisco.aci.aci_epg_useg_attribute_simple_statement:
```

Fix the below examples